Gopkg.toml

```diff
@@ -17,18 +16,27 @@
 #   source = "github.com/myfork/project2"
 #
 # [[override]]
-#  name = "github.com/x/y"
-#  version = "2.4.0"
+#   name = "github.com/x/y"
+#   version = "2.4.0"
+#
+# [prune]
+#   non-go = false
+#   go-tests = true
+#   unused-packages = true
 
 
 [[constraint]]
   name = "github.com/aws/aws-sdk-go"
-  version = "1.12.74"
+  version = "1.13.59"
 
 [[constraint]]
   name = "github.com/google/uuid"
   version = "0.2.0"
 
+[[constraint]]
+  name = "github.com/gruntwork-io/terratest"
+  version = "0.9.4"
```

Comment:
Wait, why does this code depend on itself?

Reply:
I was wondering that too... The file was auto generated by `dep` so it wasn't a conscious decision but it might be worth investigating.

Reply:
Does something import the root `terratest` folder in the code?

Reply:
No

Reply:
Hm, I have no clue. Perhaps it's confused by the folder structure, where there are no `.go` files in the root, and only in `modules/xxx` subfolders? Not sure how to fix that, other than manually removing this, as it will likely cause issues.

Reply:
Sorry, it was my bad. I was working on my fork in `$GOPATH/src/github.com/mastertinner/terratest`. That lead to this import statement. It's now fixed.

Reply:
Sorry, it was my bad. I was working on my fork in `$GOPATH/src/github.com/mastertinner/terratest`. That lead to this import statement. It's now fixed.

Reply:
Ah, OK, great 👍